src/components/procedure/_remove-ruleset.mdx

```diff
@@ -1,15 +1,22 @@
-To **disable a ruleset** from the Policies page:
+You can disable a specific rule or ruleset to prevent Semgrep Code from using it when scanning your codebase.
 
-1. In Semgrep Cloud Platform, click **Rules > [Policies](https://semgrep.dev/orgs/-/policies)**.
+:::info
+- When you disable a rule, all previous findings generated by that rule are still retained.
+:::
+
+<details>
+<summary>Disable a ruleset using the <b>Policies</b> page</summary>
+To <b>disable a ruleset</b> using the Policies page:
+
+1. In Semgrep Cloud Platform, click <b>Rules > [Policies](https://semgrep.dev/orgs/-/policies)</b>.
 2. From the **Ruleset** drop-down box, click the ruleset to remove.
 3. Click the <i class="fas fa-check-square"></i> **Matching rules**.
 4. Click **Change modes > Disabled**.
 
-:::info
-- When you disable a rule, all previous findings generated by that rule are still retained.
-:::
+</details>
 
-You can also **disable a rule** through the Findings page in both **Group by rule** and **No grouping** views. 
+<details>
+<summary>Disable a rule using the <b>Findings</b> page while in <b>Group by rule</b> view</summary>
```

Comment:
I think this section needs to be updated (lines 23-27) as it is no longer accurate - I tried to go to the details of a finding, but there's no more "Disable rule"
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/647b7730-bddc-46bb-a9a1-c1e9deb4f04a)

Reply:
:sob: Let's go with yours, as your deployment is more recent.